File: dsa/array_string_obj/isArmstrongNumber.js
let n=153,r,sum=0,temp;
temp=n;
while(temp>0){
    r=temp%10;
    sum=sum+r*r*r;
    temp=Math.floor(temp/10);
}
if(n==sum){
    console.log(n,' is armstrong number')
}else{
    console.log(n,' is not armstrong number')
}

File: dsa/array_string_obj/isArmstrongNumber.py
n = 153
sum = 0
temp = n

while temp > 0:
    r = temp % 10
    sum += r ** 3
    temp //= 10

if n == sum:
    print(f"{n} is an Armstrong number")
else:
    print(f"{n} is not an Armstrong number")
